tests/test_cli.py

```diff
@@ -363,15 +411,6 @@ def test_run_empty_file(self):
     def test_run_non_ascii_file(self):
         path = os.path.join(self.wd, 'non-ascii', 'éçäγλνπ¥', 'utf-8')
 
-        # Make sure the default localization conditions on this "system"
-        # support UTF-8 encoding.
-        loc = locale.getlocale()
-        try:
-            locale.setlocale(locale.LC_ALL, 'C.UTF-8')
-        except locale.Error:
-            locale.setlocale(locale.LC_ALL, 'en_US.UTF-8')
-        self.addCleanup(locale.setlocale, locale.LC_ALL, loc)
-
```

Comment:
So because we are using `setlocale` in `cli.run` with `C.UTF-8` as the default, those lines are not needed anymore. The command will fail if `C.UTF-8` is not available - and UTF-8 is enabled for sure. So this test should be good without those lines.

Also note that `getlocale()` + the cleanup don't do what you expect them to do. `getlocale()` does not return `LC_ALL`, but `LC_CTYPE`, I think, and the cleanup could end up setting the locale to something different as it was before.